packages/react/src/themes/teams/getBorderFocusStyles.ts

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react'
+
+import { ICSSInJSStyle, SiteVariablesPrepared } from '../types'
+
+type CSSBorderStyles = Pick<React.CSSProperties, 'borderWidth' | 'borderRadius'>
+
+type BorderFocusStyles = CSSBorderStyles & {
+  siteVariables?: SiteVariablesPrepared
+  focusInnerBorderColor?: string
+  focusOuterBorderColor?: string
+  isFromKeyboard?: boolean
+}
+
+type BorderPseudoElementStyles = CSSBorderStyles & { borderEdgeValue: string }
+
+const defaultColor = 'transparent'
+
+const getPseudoElementStyles = (args: BorderPseudoElementStyles): ICSSInJSStyle => {
+  const { borderEdgeValue, ...styles } = args
+
+  return {
+    content: '""',
+    position: 'absolute',
+    borderStyle: 'solid',
+    top: borderEdgeValue,
+    right: borderEdgeValue,
+    bottom: borderEdgeValue,
+    left: borderEdgeValue,
+    ...styles,
+  }
+}
+
+/**
+ * Returns style object that can be used for styling components on focus state
+ * NOTE: the element where this is used needs to have relative positioning so that the
+ * pseudo elements created on focus can be properly positioned
+ */
+const getBorderFocusStyles = (args: BorderFocusStyles): ICSSInJSStyle => {
+  const sv = args.siteVariables || ({} as BorderFocusStyles)
```

Comment:
cast to `BorderFocusStyles` is semantically incorrect in this context. Lets cast to the type of `siteVariables`, i.e. to `SiteVariablesPrepared`. Also, braces may be safely omitted

Reply:
fixed, thanks